config/cloud.cfg.tmpl

```diff
@@ -102,10 +102,10 @@ cloud_init_modules:
 {% if variant in ["alpine", "photon"] %}
  - resolv_conf
 {% endif %}
-{% if not variant.endswith("bsd") %}
-{% if variant not in ["photon"] %}
+{% if variant in ["alpine", "debian", "fedora", "freebsd", "rhel", "ubuntu"] %}
  - ca-certs
 {% endif %}
+{% if not variant.endswith("bsd") %}
  - rsyslog
```

Comment:
really need to fix this one…

Reply:
what's the issue?

Reply:
time, and brain
https://bugs.launchpad.net/cloud-init/+bug/1798055

Reply:
OK but whether or not rsyslog should be enabled for any BSD is unrelated to this PR, I kept the existing logic in this regard.

Reply:
yeah, sorry.
that comment was for me, really, not for you.